mobile/src/modules/auth/screens/HomeScreen.tsx

```diff
@@ -57,6 +62,8 @@ class Home extends React.PureComponent<TProps, State> {
   }
 
   componentDidMount () {
+    this.selectCurrency = this.selectCurrency.bind(this);
```

Comment:
just use arrow functions instead of bind

```
selectCurrency = () => { ... }
```

Reply:
good call